common/styles/utilities/_root-scope-classes.scss

```diff
@@ -403,18 +403,20 @@
   max-width: 100%;
 }
 
-// These classes should probably become something generic
-// that we can look at when we look at having more breakout type content.
-.person-image {
-  max-width: 100px;
+// Images
+.width-inherit {
+  width: inherit;
 }
 
-.half-in-image {
-  margin-top: -50px;
-}
+.avatar {
+  width: 78px;
+  height: 78px;
+  clip-path: inset($border-radius-unit round $border-radius-unit);
```

Comment:
Don't think this'll work in IE/Edge? Could you border-radius + overflow: hidden the wrapper instead?

Reply:
Yea, I was going to say that IE doesn't get this magic...

We use clip-path on the images too - or do you think this integral / worth the effort to get it working in IE?